WalletWasabi.Tests/Helpers/WabiSabiFactory.cs

```diff
@@ -344,6 +344,8 @@ public static (IKeyChain, SmartCoin, SmartCoin) CreateCoinKeyPairs(KeyManager? k
 			"CoinJoinCoordinatorIdentifier",
 			coinSelector,
 			new LiquidityClueProvider(),
+			0.3m,
+			150.0m,
```

Comment:
I really would prefer to use satoshis/vbytes because that's the real rate.

Reply:
I do not understand this comment. The rate, 150, is in satoshis/vbytes so I don't understand what you want to change here